src/test/groovy/edu/oregonstate/mist/locations/frontend/MockUriInfo.groovy

```diff
@@ -1,50 +1,106 @@
 package edu.oregonstate.mist.locations.frontend
+
 import javax.ws.rs.core.UriInfo
 import javax.ws.rs.core.UriBuilder
 import javax.ws.rs.core.PathSegment
 import javax.ws.rs.core.MultivaluedMap
 import javax.ws.rs.core.MultivaluedHashMap
 
 class MockUriInfo implements UriInfo {
```

Comment:
We have two copies of this class now. One in src/test/groovy/edu/oregonstate/mist/api, and one in src/test/groovy/edu/oregonstate/mist/locations/frontend.